index.html

```diff
@@ -310,6 +310,17 @@ <h3>Authorization</h3>
 or legacy authorization technologies.
       </p>
 
+      <section>
+        <h4>Delegating Authorization</h4>
+        <p>
+Implementers SHOULD utilize authorization protocols that support authorization
```

Comment:
```suggestion
Implementers MAY utilize authorization protocols that support authorization
```

Reply:
I am supportive of the change to MAY because we don't have a good production way of enforcing the SHOULD in testing

Reply:
Some SHOULDs are not mechanically testable. That's an unfortunate reality. It does not change the fact that this guidance SHOULD be SHOULD (an admonition to act a certain way), not MAY (simple permission to act that way).